packages/core/useScroll/index.ts

```diff
@@ -251,11 +259,15 @@ export function useScroll(
   )
 
   tryOnMounted(() => {
-    const _element = toValue(element)
-    if (!_element)
-      return
-
-    setArrivedState(_element)
+    try {
+      const _element = toValue(element)
```

Comment:
```suggestion
      const _element = unrefElement(element)
```

Reply:
Could be the root of the problem 🤔

Reply:
Yes. Sorry for the lack of explanation.

The problem occurs because the `ref` value itself is a component instance when it's being used in components (refer to [documentation](https://vuejs.org/guide/essentials/template-refs.html#ref-on-component)), so the `toValue` is just unwrapping it (as expected). Since  `getComputedStyle` doesn't exist on the instance object, an error is thrown.

Luckily, VueUse already have something to handle this, the `unrefElement` that I used in the suggestion.